backend/uclapi/oauth/templates/permissions.html

```diff
@@ -0,0 +1,28 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html>
+<head>
+
+  <title>UCL API Authorisation</title>
+
+	<link rel="stylesheet" href="https://unpkg.com/purecss@0.6.2/build/pure-min.css" integrity="sha384-UQiGfs9ICog+LwheBSRCt1o5cbyKIHbwjWscjemyBMT9YCUMZffs6UqUTd0hObXD" crossorigin="anonymous">
+	<link rel="stylesheet" href="https://unpkg.com/purecss@0.6.2/build/grids-responsive-min.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <link rel="stylesheet" href="{% static "css/dashboard.css" %}" type="text/css" />
+
+</head>
+
+
+<body>
+
+	<script>
+    window.initialData = {{ initial_data|safe }};
+    window.staticURL = {% static "" %}
+  </script>
+	<div class="app"></div>
+  <div class="csrfDiv">
+  <script src="{% static "js/authorise.js" %}"></script>

```

Comment:
the name of the template and JavaScript file should probably have the same name.